modules/lambda-function/outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "arn" {
+  value       = "${aws_lambda_function.compute_queue_backlog.arn}"
+  description = "This is compute queue backlog lambda's unqualified arn."
+}
+
+output "name" {
+  value       = "${aws_lambda_function.compute_queue_backlog.function_name}"
+  description = "This is compute queue backlog lambda's function name."
+}
+
+output "execution_role_arn" {
```

Comment:
If you pass in an execution role and one isn't created, this will be empty or error (not sure which)

Reply:
I mention this in the description. It will not error. It will be empty, which is by design.